documentation/docs/specs/did/iota_did_method_spec.md

```diff
@@ -14,290 +14,210 @@ keywords:
+The following steps can be used to read the latest DID document associated with a DID.
+1. Obtain the `Alias ID` from the DID by extracting the `iota-tag` from the DID, see [DID Format](#did-format).
+2. Query the Alias Output corresponding to the `Alias ID`, using a node running the [inx indexer](https://github.com/iotaledger/inx-indexer). Nodes usually include this indexer by default.
+3. Retrieve the value of the `State Metadata` field from the returned output.
+4. Check if its content matches the [Anatomy of the State Metadata](#anatomy-of-the-state-metadata). Return an error otherwise.
+5. Decode the DID document from the state metadata.
+6. Replace the placeholder `did:0:0` with the actual DID which can be derived from the `Alias ID` of the output.
 
 ### Update
 
-In order to update a DID Document, either an Integration or a Diff DID message needs to be generated. It is RECOMMENDED to use only Integration DID messages if the DID Document is updated very infrequently and it is expected to never go beyond 100 updates in the lifetime of the DID. If that is not the case, it is RECOMMENDED to use as many Diff DID messages instead with a maximum of around 100 updates per Diff chain. 
-
-#### Creating an Integration DID message
-
-An Integration DID message is unrestricted in its operations and may add or remove any fields to the DID Document. In order to query the DID Document, every Integration DID message must be processed, therefore it is RECOMMENDED to reduce the usage of these messages unless the DID Document is updated very infrequently. 
+Updating a DID Document can be achieved by a state transition of the Alias Output. A state controller can update the
+`State Metadata` in the Alias Output which reflects an update to the DID Document.
 
-In order to create a valid Integration DID message, the following steps are to be performed:
-1. Create a new DID Document that contains all the new target state. This SHOULD include the new desired changes and all the changes inside the previous Diff Chain, otherwise these changes are lost!
-2. Retrieve the IOTA MessageId from the previous Integration DID message and a keypair for signing the DID Document.
-3. Set the `previousMessageId` field to the IOTA MessageId value.
-4. Create a `proof` object referencing the public key and signature suite from a verification method in the DID Document able to sign updates as per the [Signing Keys](#signing-keys) section.
-5. Create a cryptographic signature using the same keypair and add the result to the appropriate field(s) inside the `proof` object.
+1. Create the content of the updated DID Document.
+2. Create the payload and the headers as described in the [Anatomy of the State Metadata](#anatomy-of-the-state-metadata).
+3. Create a new Alias Output with the same `Alias ID` as the current output.
+4. Increment the `State Index`.
+5. Leave the unlock conditions unchanged.
```

Comment:
That was my suggestion, but it's actually incorrect. It's perfectly fine to update the unlock conditions during a DID update. Again, I would love to just leave this point out because this is not our concern, it's the concern of output validation, i.e. TIP-18. No matter if we write this, or "Set state controller and governor" like in _Deactivation_, we're stating incorrectly what is actually allowed. Technically, the user can do whatever is valid according to TIP-18.

The correct way would be to say something like "Set valid unlock conditions." at which point we're not saying anything helpful and might as well leave it out.

As discussed, it's fine to leave it as-is for now and address it later, but we should address it at some point.

Reply:
Why is this incorrect? state transition doesn't allow changing controllers.

Reply:
True that. Sorry I got confused on that part :man_shrugging:.